pkg/manager/manager_test.go

```diff
@@ -970,10 +970,15 @@ var _ = Describe("manger.Manager", func() {
 			Expect(err).NotTo(HaveOccurred())
 			Expect(resp.StatusCode).To(Equal(http.StatusOK))
 
-			// Check readiness path without trailing slash
+			// Check readiness path without trailing slash without redirect
 			readinessEndpoint = fmt.Sprint("http://", listener.Addr().String(), strings.TrimSuffix(defaultReadinessEndpoint, "/"))
```

Comment:
`defaultReadinessEndpoint` no longer ends in `/` so no need to trim it:
```suggestion
			readinessEndpoint = fmt.Sprint("http://", listener.Addr().String(), defaultReadinessEndpoint)
```

Reply:
Thanks for your kind review ! Totally agree with you 😆 
I addressed 👍 

https://github.com/kubernetes-sigs/controller-runtime/compare/c9ee1568f928ed47200aa0faedd867b4f60153c2..d6fcdd62cf2db5e2b208fb7c32d05cbecfa83999